packages/eslint-plugin/src/rules/dollar-sign.js

```diff
@@ -0,0 +1,24 @@
+/** @type {import('eslint').Rule.RuleModule} */
+export default {
+  meta: {
+    type: 'suggestion',
+    docs: {
+      description: 'Example rule that checks for dollar signs',
+      recommended: true,
+    },
+    schema: [], // no options
+  },
+
+  create(context) {
+    return {
+      Identifier(node) {
+        if (node.name.includes('$')) {
+          context.report({
+            node,
+            message: 'Avoid using $ in identifiers',
```

Comment:
Is the dollar-sign avoidance an orthogonal improvement, or are these somehow coupled?

Reply:
it was part of the scaffold. a subsequent commit removed it